### DISCUSSION TITLE
Joomla Install 3.10.11

### DISCUSSION THREAD

**Discussion by cosmoelton** [joomla/joomla-cms#39406] [General]
Olá, Guys.

Estou instalando o joomla 3.10.11 com php 8.0, depois da configurações iniciais, fica parado nessa tela.

não tem nada nos logs do apache2

![image](https://user-images.githubusercontent.com/116853196/207223831-8810f23a-6ab8-4067-a793-aaffb59ddde7.png)

**Comment by richard67** [comment]
Read the content of that screen. The installation has finished with success. You have to use the button to remove the "installation" folder before you can continue. When you have done that, you can use the buttons to go to the site (frontend) or to the administrator (backend).

**Reply by richard67** [reply]
If that doesn't work for you, check the permissions of the folder named "Installation" in your Joomla root folder.

**Reply by cosmoelton** [reply]
I already solved it by manually removing the installation folder, Thank you
